src/routes/rules/RulesView.vue

```diff
@@ -411,18 +388,35 @@ export default {
     this.royals = royals;
     this.oneOffs = oneOffs;
     this.sectionTitles = sectionTitles;
-  },
-  methods: {
-    goToSection(url) {
-      this.goTo(url, this.scorllOptions);
-      window.location.hash = url;
-    },
-    onIntersect(isIntersecting, entries) {
+
+    // Scrolling
+    this.scrollOptions = {
+      duration: 1000,
+      offset: -100,
+      easing: 'easeInOutCubic',
+    };
+
+    // Intersection
+    const onIntersect = (_isIntersecting, entries) => {
       entries.forEach((entry) => {
         if (entry.isIntersecting) {
           this.activeTitle = entry.target.id;
         }
       });
+    };
+
+    this.intersectConfig = {
+      handler: onIntersect,
+      options: {
+          threshhold: 0,
+          rootMargin: '-150px 0px -300px 0px',
```

Comment:
This is the crux of the fix. It makes it so the element only counts as intersecting when at least 150px below the top of the viewport and 300 above the bottom

Reply:
Will this still work correctly on big screens? Should it use a percentage?

Reply:
Negative %'s aren't supported, so I think we are best off sticking with px. My hope is that this ensures it's at least fairly far onto the screen. I'll bump the threshold to .2 so that it also requires at least 20% of the section to be visible, which I think will further mitigate the possibility of it triggering prematurely on large screens (though it may not be perfect).